<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyView</class>
 <widget class="QWidget" name="StudyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="ClickLabel" name="clk_lb_coffee">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Drink coffee</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ScaleLabel" name="clk_lb_desk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_writer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Writer does...</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_wine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Drink wine</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_books">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Write book</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ScaleLabel" name="sc_lb_study">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_sofa">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Sleep</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ScaleLabel" name="clk_lb_window">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_apple">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Eat</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_writer_rhand">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_writer_lhand">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="sc_lb_writer_head_left">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="sc_lb_writer_head_right">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="clk_lb_computer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Writer does...</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ScaleLabel" name="sc_lb_light">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ScaleLabel" name="sc_lb_weather">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ScaleLabel" name="sc_lb_weather_alternate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ScaleLabel" name="sc_lb_coffe_steam">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>sc_lb_weather</zorder>
  <zorder>sc_lb_study</zorder>
  <zorder>clk_lb_sofa</zorder>
  <zorder>clk_lb_desk</zorder>
  <zorder>clk_lb_writer</zorder>
  <zorder>clk_lb_books</zorder>
  <zorder>clk_lb_coffee</zorder>
  <zorder>clk_lb_window</zorder>
  <zorder>clk_lb_apple</zorder>
  <zorder>clk_lb_writer_rhand</zorder>
  <zorder>clk_lb_writer_lhand</zorder>
  <zorder>clk_lb_computer</zorder>
  <zorder>sc_lb_writer_head_left</zorder>
  <zorder>sc_lb_writer_head_right</zorder>
  <zorder>sc_lb_weather_alternate</zorder>
  <zorder>sc_lb_light</zorder>
  <zorder>clk_lb_wine</zorder>
  <zorder>sc_lb_coffe_steam</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>view/clicklabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>ScaleLabel</class>
   <extends>QLabel</extends>
   <header>view/scalelabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
